services/ui-src/src/verbiage/pages/accordion.ts

```diff
@@ -147,10 +147,7 @@ export default {
           content: "<b>Admin instructions</b>",
         },
       ],
-      list: [
-        "To allow a state to make corrections or edits to a submission use “Unlock” to release the submission. The status will change to “In revision”.",
-        "Submission count is shown in the # column. Submissions started and submitted once have a count of 1. When a state resubmits a previous submission, the count increases by 1.",
-      ],
```

Comment:
What was the reason for this change?

Reply:
I originally had that text copied from the workplan. Since we don't have guidance yet for what the instructions will be for ABCD I figured it made more sense to replace the text with more of a placeholder.